### PR TITLE
KBC-1336 Configuration rollback in dev branch

### PR DESCRIPTION
**Pull request by zajca**
Jira: KBC-1336 
KBC: https://github.com/keboola/connection/pull/3102

Before asking for review make sure that:

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---

### CODE REVIEW

File: tests/Common/BranchComponentTest.php [hunk 1]
```diff
@@ -1356,6 +1356,81 @@ public function testVersionIncreaseWhenUpdate()
...
+
+        // update config - conf V5
+        $componentsApi->updateConfiguration($configuration->setConfiguration(['d' => 'b']));
+
+        // wait a moment, rollbacked version should have different created date
+        sleep(2);
+
+        // rollback to version 2 - conf V6
+        // second row should be missing, and first row should be rolled back to first version
+        $componentsApi->rollbackConfiguration('wr-db', $configurationV1['id'], 2);
```
Comment (fejjo) [inline tests/Common/BranchComponentTest.php:1403]: Otestuj prosim este ak zadam custom `changeDescription`

**Review by fejjo — CHANGES_REQUESTED** [review]
Jedna drobnost

File: tests/Common/BranchComponentTest.php (hunk 1 above)
Comment (zajca) [reply tests/Common/BranchComponentTest.php:1403]: přidáno

**Review by fejjo — APPROVED** [review]
👍

**Pull request merged**